src/markdoc/tags/Cards_Item.svelte

```diff
@@ -3,17 +3,31 @@
 
     export let href: string;
     export let icon = '';
+    export let image = '';
+    export let dark = '';
```

Comment:
Let's use the light/dark theme markdoc component we already have in place ✌️

Reply:
Done